content/reference-capabilities/combining-capabilities.md

```diff
@@ -8,15 +8,17 @@ menu:
 toc: true
 ---
 
-When a field of an object is read, its reference capability depends both on the reference capability of the field and the reference capability of the __origin__, that is, the object the field is being read from.
+When we talked about fields in the [classes]({{< relref "types/classes.md" >}}) and [variables]({{< relref "expressions/variables.md" >}}) chapters, we passed over the detail of field capabilities. Fields, just like variables, have their own capabilities! A field in a class has to make the exact same guarantees. An `iso` field, for instance, is globally unique. The only way to reference it is through this single field of the object (except of course, `tag` references, or shared `val` portions).
```

Comment:
Still trying to figure out the exact wording, but I think the above is a tiny bit too simple for my taste. One point I do not want to leave ambiguous is the nature of global uniqueness when the origin capability isn't necessary unique. I.e. the intuition around an `iso` field in a `ref` object, which is an interaction that readers won't have seen before with local variable

Reply:
That is a fair concern. Balancing the wording to not suggest the incorrect conclusion is difficult. Do you have any thoughts on how the wording might change?

Reminder that the audience for the Tutorial is:

> ...aimed at people who have some experience programming already. It doesn’t really matter if you know a little Python, or a bit of Ruby, or you are a JavaScript hacker, or Java, or Scala, or C/C++, or Haskell, or OCaml: as long as you’ve written some code before, you should be fine.

Therefore I think it is fine if you add complexity based on more common knowledge for someone who has been programming a few years. The Tutorial should only be as simple as it needs to be, but how simple is too simple becomes a matter of opinion fast. I think it would be fair to perhaps leverage an example based around scoping rules, but unfair to base an example around recursive patterns -- the former is more universal than familiarity with the latter.

Let me know if I can help refine the wording here!

Reply:
The thing I think I want to get at with the wording is that the unit of uniqueness here is this one field of this concrete object, independent of all the references to that object. Now, with the curse of knowledge it's difficult to figure out exactly what to address, but the main point I feel is critical to this description is making sure we describe _what_ is unique and is the unit of ownership, and so that's why I had the not perfect wording:

>  The only way to reference it is through this single field of the object